crypto/rsa/rsa_lib.c

```diff
@@ -286,6 +393,18 @@ void RSA_set_flags(RSA *r, int flags)
     r->flags |= flags;
 }
 
+int32_t RSA_get_version(RSA *r)
+{
+    /* { two-prime(0), multi(1) } */
+    return r->version;
+}
+
+void RSA_set_version(RSA *r, int multi)
```

Comment:
That's because parameter `multi` is a boolean value, it's set directly to `r->version`

Reply:
Right - but why are we doing that? In my head I would think that either they should both use a boolean, or both use the actual version number (in which case RSA_set_version() should return an error if you attempt to set an unsupported version). I tend to think the latter might be better in case there is ever another version of the RSA structure that we want to support.

Reply:
If it isn't get/set of a version then the function should be named differently IMHO.

Reply:
Changed to stay consistency with the RSA_get_version, and also two extra macros defining the version numbers are exported.